apps/prairielearn/src/admin_queries/PLR_start.json

```diff
@@ -1,4 +1,4 @@
 {
-  "description": "Create 14 mock users, put them in a live session, give them random scores and times",
+  "description": "Starts a PLR live session",
```

Comment:
Later on we should accept parameters into here so that we can start a live session based off a specific ID. Generally having more versatile admin queries is nice.

Reply:
I can do that but I feel like that should be a pr of it's own. Only reason I built the admin query in this one was so that I could directly test it.